src/app/search/search-filters.ts

```diff
@@ -1112,17 +1119,12 @@ function searchFilters(
       },
       powerlimit(item: D2Item, predicate: string) {
         // anything with no powerCap has no known limit, so treat it like it's 99999999
-        return mathCheck.test(predicate) && compareByOperator(item.powerCap ?? 99999999, predicate);
-        // hypothetically we can use this mathcheck to divert if we decided to support something like "powerlimit:arrivals"
+        return compareByOperator(item.powerCap ?? 99999999, predicate);
       },
       sunsetsafter(item: D2Item, predicate: string) {
         const itemFinalSeason = getItemPowerCapFinalSeason(item);
-        return (
-          itemFinalSeason &&
-          mathCheck.test(predicate) &&
-          compareByOperator(itemFinalSeason, predicate)
-        );
-        // hypothetically we can use this mathcheck to divert if we decided to support something like "sunsetsafter:arrivals"
+        const interpretedPredicate = predicate.replace(/[a-z]+$/i, (tag) => seasonTags[tag]);
```

Comment:
Can you memoize this or move it into the query parsing phase? This'll run for every item.

Reply:
ooh actually. i thought there wasn't a good place for it, but there sort of is, until we revamp filters.